requirements/docs.txt

```diff
@@ -1,7 +0,0 @@
-sphinx>=1.4.9
```

Comment:
Where do the docs requirements come from now?

Reply:
Under [dev-packages] in the Pipfile - is this okay?

Reply:
I have re-added this file as discussed.